frontend/src/pages/Guests/NewGuest/NewGuest.tsx

```diff
@@ -258,7 +258,7 @@ const NewGuest = (props: RouteComponentProps) => {
       const stayData = {
         guest_id: userId,
         house_id: houseId,
-        extra_guests: extraGuests,
+        extra_guests: +extraGuests,
```

Comment:
Just wanted to double-check this: did you mean to prepend the `+` to this here?

Reply:
Yes 👍 
It coerces it into a number which is what we need.